backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label Spree::StockLocation.model_name.human %>
```

Comment:
Will go with correct code and wait for the select2 fix.  that seems like the most correct option short of fixing select2 and submitting them a PR.